cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ConstraintsTest.scala

```diff
@@ -59,15 +59,22 @@ class ConstraintsTest extends DocumentingTestBase with SoftReset {
       title = "Create a unique constraint",
       text = "When creating a unique constraint, a name can be provided. The constraint ensures that your database " +
         "will never contain more than one node with a specific label and one property value.",
-      queryText = "CREATE CONSTRAINT constraint_name ON (book:Book) ASSERT book.isbn IS UNIQUE",
+      queryText = "CREATE CONSTRAINT constraint_name FOR (book:Book) REQUIRE book.isbn IS UNIQUE",
       assertions = _ => assertConstraintWithNameExists("constraint_name", "Book", List("isbn"))
     )
+    testQuery(
+      title = "Create a unique constraint using deprecated syntax",
+      text = "The unique constraint ensures that your database " +
+        "will never contain more than one node with a specific label and one property value.",
```

Comment:
```suggestion
        "never contains more than one node with a specific label and one property value.",
```
We like to use present tense and active voice, but I am aware that this has not been consistently implemented, and not in this section either. Perhaps it is better to leave it as is, for consistency's sake.

Reply:
Yes, I think this is just copy paste from another test in the same file, so perhaps it is better to make a cleanup card on either your Trello board our ours about updating it across the board.

Reply:
I agree, it's best dealt with separately. And with that, I have no more objections. Should I merge?

Reply:
please do :)